docker/standalone.xml

```diff
@@ -132,7 +130,7 @@
             </formatter>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:bean-validation:1.0"/>
-        <subsystem xmlns="urn:jboss:domain:datasources:5.0">
+        <subsystem xmlns="urn:jboss:domain:datasources:4.0">
```

Comment:
I found this change in the generated `distribution/server-dist/target/keycloak-3.2.0.Final/standalone/configuration/standalone.xml`